src/util/util.spec.ts

```diff
@@ -49,3 +49,26 @@ describe('isSubstringOf', () => {
     expect(isSubstringOf('', '')).toBeTruthy()
   })
 })
+
+describe('isError guard function', () => {
+  describe.each`
+    arg                      | expectedResult
+    ${{ name: 'Not error' }} | ${false}
+    ${{ name: 'error' }}     | ${false}
+    ${{ name: 'Error' }}     | ${false}
+    ${{}}                    | ${false}
+    ${true}                  | ${false}
+    ${false}                 | ${false}
+    ${'test string'}         | ${false}
+    ${new TypeError()}       | ${false}
+    ${new SyntaxError()}     | ${false}
```

Comment:
Following the comment about the `isError` function implementation, I think most of these test cases are unnecessary, and the last one must provide an expected result to `true`.

Reply:
I think it's important to test negative cases as well though so I'd keep some test cases